### ISSUE TITLE
As a marketplace vendor, I don't want Magfest to submit the wrong info to the comptroller

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem#3062]
Ask Kator for details,

For some vendors, we're submitting individual names instead of business names to the MD comptroller.  Probably need to review our data collection and make it clearer if possible to the vendors when they sign up what info gets sent to the comptroller and whether to use their names or business names for certain fields.

**Comment by binary1230** [comment]
(for next magfest)

**Comment by binary1230** [comment]
(see convos in #office on Slack around Dec 22nd for more context.  the above is basically it)

**Comment by RobRuana** [comment]
We probably just need a field like this:

<img width="950" alt="screen shot 2017-12-22 at 5 22 07 pm" src="https://user-images.githubusercontent.com/2592431/34313887-b48212a4-e73c-11e7-8162-f75905aa0a14.png">

**Comment by kitsuta** [comment]
Yeah, agreed that the extra field would probably go a long way towards getting better data.
